bom/pom.xml

```diff
@@ -717,16 +717,6 @@
                             <versionString>${version.org.hibernate}</versionString>
                         </configuration>
                     </execution>
-                    <execution>
-                        <id>parse-search-version</id>
-                        <goals>
-                            <goal>parse-version</goal>
-                        </goals>
-                        <configuration>
-                            <propertyPrefix>parsed-version.org.hibernate.search</propertyPrefix>
-                            <versionString>${version.org.hibernate.search}</versionString>
-                        </configuration>
-                    </execution>
```

Comment:
Please don't remove this. Especially since you state that your change is temporary.

Reply:
I'm going to put it pack.